server/sonar-web/src/main/js/components/issue/collections/issues.js

```diff
@@ -31,11 +31,11 @@ export default Backbone.Collection.extend({
   _injectRelational (issue, source, baseField, lookupField) {
     const baseValue = issue[baseField];
     if (baseValue != null && _.size(source)) {
-      const lookupValue = _.find(source, function (candidate) {
+      const lookupValue = _.find(source, candidate => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)